.github/workflows/pkgci_regression_test.yml

```diff
@@ -298,16 +298,16 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           pytest SHARK-TestSuite/iree_tests/benchmarks/sdxl/benchmark_sdxl_rocm.py \
-            --goldentime-rocm-e2e-ms 1661.5 \
-            --goldentime-rocm-unet-ms 450.5 \
-            --goldentime-rocm-clip-ms 19 \
+            --goldentime-rocm-e2e-ms 1335.0 \
+            --goldentime-rocm-unet-ms 340.0 \
+            --goldentime-rocm-clip-ms 17.0 \
             --goldentime-rocm-vae-ms 290.0 \
```

Comment:
These might be tight. Presubmit got:
* 1334 (-1 from 1335)
* 339 (-1 from 340)
* 16.3 (-0.7 from 17.0)
* 290.0 (-0 from 290.0)

Reply:
unet and clip are pretty stable usually so that should be fine. vae and e2e might have a bit more variation though, so might want to add a bit more leniency there (1336 and 291?)